ocaml/testsuite/tests/warnings/w53.ml

```diff
@@ -355,3 +355,21 @@ module TestNewtypeAttr = struct
 
   let f2 = fun [@boxed] (type a) (x : a) -> x (* rejected *)
 end
+
+module type TestErrorMessageSig = sig
+  type 'a t1 = 'a [@@error_message ""] (* rejected *)
+  type s1 = Foo1 [@error_message ""] (* rejected *)
+  val x : int [@@error_message ""] (* rejected *)
+
+  external y : (int [@error_message ""]) -> (int [@error_message ""]) = "x" "y" (* rejected *)
+  external z : int -> int = "x" "y" [@@error_message ""] (* rejected *)
+end
+
+module TestErrorMessageStruct = struct
+  type 'a t1 = 'a [@@error_message ""] (* rejected *)
+  type s1 = Foo1 [@error_message ""] (* rejected *)
+  let x : int = 42 [@@error_message ""] (* rejected *)
+
+  external y : (int [@error_message ""]) -> (int [@error_message ""]) = "x" "y" (* rejected *)
+  external z : int -> int = "x" "y" [@@error_message ""] (* rejected *)
+end
```

Comment:
After redoing the code in Builtin_attributes, a repeated `[@error_message ...]` should trigger warning 53, so add a test here.

Reply:
The layout annotation feature is behind `-extension layouts`. Not sure if we want to add that flag to this file. Added tests to `error_message_attr_w53.ml` with a CR about moving it to `w53.ml`.

Reply:
Great.